src/model/template.paths.ts

```diff
@@ -18,4 +18,5 @@ export enum Templates {
   ERROR = "error",
   FILE_ERROR = "file-error",
   ACCESSIBILITY_STATEMENT = "accessibility-statement",
+  ERROR_404 = "page-not-found.html"
```

Comment:
the other names don't seem to have '.html' on them, will it work without?

Reply:
Don't need the html part - removed.